salt/modules/selinux.py

```diff
@@ -604,3 +626,99 @@ def fcontext_apply_policy(name, recursive=False):
                 del new[key]
             ret['changes'].update({filespec: {'old': old, 'new': new}})
     return ret
+
+
+def port_get_policy(name, sel_type=None, protocol=None, port=None):
+    '''
+    .. versionadded:: develop
```

Comment:
This should be Fluorine.

Reply:
Should it be set to literally "Fluorine"? https://docs.saltstack.com/en/latest/topics/releases/version_numbers.html shows the version number as `TBD` - I wasn't sure how/when these get updated when released.

Reply:
Yes, it should be just the word `Fluorine` we do a search and replace on the code word once we know when the release date will be.

Reply:
Thanks, updated